bin/utils/print-joblog-to-file.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/sh
+#######################################################################
+# This program and the accompanying materials are made available
+# under the terms of the Eclipse Public License v2.0 which
+# accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Copyright Contributors to the Zowe Project. 2020
+#######################################################################
+
+# Function: Print a JES job log to a USS file
+#
+# Inputs  - jobname   - the name of the job
+#         - jobid     - JES JobID of job, like job00000, stc00000 or tsu00000
+# Lowercase is treated as uppercase in jobname and jobid
+#
+# Outputs - jobfile   - the USS filename to receive the output
+#
+# The job will be purged from the JES spool
+# Uses    - The TSO OUTPUT command
+
+# identify this script
+SCRIPT_DIR="$(dirname $0)"
+SCRIPT="$(basename $0)"
+echo script $SCRIPT started from $SCRIPT_DIR
+
+if [[ $# -ne 3 ]]
+then
+echo; echo $SCRIPT Usage:
+cat <<EndOfUsage
+
+    $SCRIPT jobname jobid jobfile
+
+for example
+
+    $SCRIPT myjob   job65262 myjob.out
+    $SCRIPT zwe1sv  stc00406 /u/userid/zowestc.out
+
+The output will be placed in jobfile and the job will be purged.
+EndOfUsage
+echo script $SCRIPT ended 
+exit 
+fi
+
+jobname=$1
+jobid=$2
+jobfile=$3
+
+userid=${USER:-${USERNAME:-${LOGNAME}}}
+tsoJobname=RUNTSOCM
+if [ -z ${TEMP_DIR+x} ]; then
```

Comment:
What is the ${TEMP_DIR+x} supposed to do? It just seems to evaluate to "x" when I try it?

Reply:
I've removed the IF at line 53.